ckanext/canada/helpers.py

```diff
@@ -774,6 +776,44 @@ def date_field(field, pkg):
     return pkg.get(field, None)
 
 
+def registry_network_access():
+    """
+    Only allow requests from GOC network to access
+    user account registration view
+    """
+    remote_addr = request.headers.get('X-Forwarded-For') or \
+                  request.environ.get('REMOTE_ADDR')
+    try:
+        client_ip = ipaddress.ip_address(text_type(remote_addr))
+    except ValueError:
+        return False
+
+    netlist_path = config.get('ckanext.canada.registry_network_list', '')
+    if not netlist_path:
+        return False
+    if not os.path.isfile(netlist_path):
```

Comment:
```suggestion
    if not os.path.isfile(netlist_path) or not os.path.isfile(netlist_path):
        return False
```

@RabiaSajjad these two conditions can just go into an OR